docs/operations/blocks-storage.template

```diff
@@ -74,7 +74,7 @@ Whenever the pool of compactors increase or decrease (ie. following up a scale u
 
 ## Index cache
 
-The querier supports a cache to speed up postings and series lookups from TSDB blocks indexes. Two backends are supported:
+The querier and store-gateway now supports a cache to speed up postings and series lookups from TSDB blocks indexes. Two backends are supported:
```

Comment:
In the documentation we shouldn't use temporarl references like "now". I would just suggest to remove it:
```suggestion
The querier and store-gateway support a cache to speed up postings and series lookups from TSDB blocks indexes. Two backends are supported:
```